packages/playground/src/app/index.js

```diff
@@ -4,7 +4,8 @@
  * Being the first import is important, so that it is the default style
  * and other style can override it
  */
-import '@talend/bootstrap-theme';
+// eslint-disable-next-line @talend/import-depth
+import '@talend/bootstrap-theme/dist/bootstrap.css';
```

Comment:
this is the first issue, today the theme is added by the CDN plugin, so now we have to add the real asset behind